addons/default/visiosoft/advs-module/src/Adv/AdvRepository.php

```diff
@@ -112,6 +113,26 @@ public function searchAdvs($type, $param = null, $customParameters = null, $limi
             $column = "JSON_EXTRACT(foreign_currencies, '$." . $param['currency'] . "') <=" . $int;
             $query = $query->whereRaw($column);
         }
+        if (!empty($param['date'])) {
+            if ($param['date'] === 'day') {
+                $query = $query->where('advs_advs.publish_at', '>=', Carbon::now()->subDay());
+            } elseif ($param['date'] === 'week') {
+                $query = $query->where('advs_advs.publish_at', '>=', Carbon::now()->subWeek());
+            } elseif ($param['date'] === 'month') {
+                $query = $query->where('advs_advs.publish_at', '>=', Carbon::now()->subMonth());
+            }
+        }
+        if (!empty($param['photo'])) {
+            $query = $query->whereNotNull('cover_photo');
+        }
+//        if (!empty($param['video'])) {
```

Comment:
do we need this? @Diatrex

Reply:
we need it for when we create the video filter @fatihalp